{{cookiecutter.project_shortname}}/site/setup.cfg

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
Overall, I would go with as little information here as possible. Anything here will need to be updated if something changes.